android/build.gradle

```diff
@@ -126,7 +126,7 @@ repositories {
 dependencies {
     //noinspection GradleDynamicVersion
   implementation "com.facebook.react:react-native:+"
-  api 'com.optimove.android:optimove-android:7.0.0'
+  api 'com.optimove.android:optimove-android:7.2.0'
```

Comment:
RE your other comment on the Android PR about keeping the bell, should this change be reverted before releasing?

Reply:
Back to 7.1.1